src/Test/Plutip/Internal/BotPlutusInterface/Setup.hs

```diff
@@ -13,7 +13,7 @@ import Cardano.Api qualified as CAPI
 import Cardano.Launcher.Node (nodeSocketFile)
 import Data.Aeson (encodeFile)
 import Data.Foldable (traverse_)
-import Plutus.V1.Ledger.Api (PubKeyHash (PubKeyHash))
+import Plutus.V2.Ledger.Api (PubKeyHash (PubKeyHash))
```

Comment:
Silly question: Does this mean that Plutip is migrating to V2 without supporting V1?

Reply:
This is under investigation

Reply:
For this case, `V2` re-exports `PubKeyHash` from `V1`.